packages/react-percy-storybook/src/getStaticAssets.js

```diff
@@ -1,16 +1,83 @@
 import path from 'path';
+import walk from 'walk';
 
 const fs = require('fs');
 
+// Some build assets we never want to upload.
+const SKIPPED_ASSETS = [
+    'index.html',
+    'iframe.html',
+    'favicon.ico',
+    'static/',
+    /\.map$/,
+    /\.log$/,
+    /\.DS_Store$/
+];
+const MAX_FILE_SIZE_BYTES = 15728640;  // 15MB.
+
+// Synchronously walk the build directory, read each file
+function gatherBuildResources(buildDir) {
```

Comment:
We could potentially move this into a shared method from percy-js since we're starting to re-use it.